samples/Samples.Server30/package-lock.json

```diff
@@ -0,0 +1,95 @@
+{
+  "name": "samples.server",
```

Comment:
This file together with package.json file can be removed from both samples. I do not write JS code but I see that this stuff is not needed here. Everything should work without it.

Reply:
Also all files in wwwroot folder

Reply:
Apparently these files appeared a long time ago and are not required for work now.